libraries/ms-drivers/inc/bts_7200_current_sense.h

```diff
@@ -57,8 +97,29 @@ StatusCode bts_7200_init_pca9539r(Bts7200Storage *storage, Bts7200Pca9539rSettin
 
 // Read the latest measurements. This does not get measurements from the storage but instead
 // reads them from the BTS7200 itself.
+// Note that, due to the fault handling implementation, the pointer to storage has to be
+// valid for BTS7200_FAULT_RESTART_DELAY_MS. Otherwise, bts_7200_stop must be called
+// before the pointer is freed to prevent segfaults.
 StatusCode bts_7200_get_measurement(Bts7200Storage *storage, uint16_t *meas0, uint16_t *meas1);
 
+// Enable output 0 by pulling the IN0 pin high.
+StatusCode bts_7200_enable_output_0(Bts7200Storage *storage);
+
+// Disable output 0 by pulling the IN0 pin low.
+StatusCode bts_7200_disable_output_0(Bts7200Storage *storage);
+
+// Enable output 1 by pulling the IN1 pin high.
+StatusCode bts_7200_enable_output_1(Bts7200Storage *storage);
+
+// Disable output 1 by pulling the IN1 pin low.
+StatusCode bts_7200_disable_output_1(Bts7200Storage *storage);
+
+// Return whether output 0 is enabled or disabled
+bool bts_7200_get_output_0_enabled(Bts7200Storage *storage);
+
+// Return whether output 1 is enabled or disabled
+bool bts_7200_get_output_1_enabled(Bts7200Storage *storage);
```

Comment:
You might consider more fully replicating the `gpio` interface - i.e. define an enum analogous to `GpioState`, define set, toggle, and get methods (each for output 0 and output 1, or maybe take that as a parameter, up to you) and take the state enum as a parameter to the set method. Up to you.